### PR TITLE
verbena-moth: Add /pricing page

### PR DESCRIPTION
**Pull request by taravancil**
This is branched off of [classy-afrovenator](https://github.com/FogCreek/Glitch-Community/pull/1155), so maybe we should wait to review/merge this until that PR is merged.

## Links
* [verbena-moth.glitch.me/pricing](https://verbena-moth.glitch.me/pricing)
* [[ch6406]](https://app.clubhouse.io/glitch/story/6406/create-a-pricing-page)

## GIF/Screenshots:
<div>

</div>

## Changes:
This adds a /pricing page behind the `pufferfish` feature flag.

## How To Test:
*Note that this is the dogfood implementation of this feature, so review should be focused on "Does it work well enough to test internally and iterate on?" rather than polish and cleverness.*

### Testing /pricing Visibility
1. Log in
2. Make sure you're in [@testing-team](https://glitch.com/@testing-team)
3. Make sure the `pufferfish` flag is enabled on [verbena-moth.glitch.me/secret](https://verbena-moth.glitch.me/secret)

| Case  | Expected Outcome | 
| ------------- | ------------- |
| In Testing Team + Enabled the feature on /secret  | /pricing `200` |
| In Testing Team + Disabled the feature on /secret  | /pricing `404` |
| NOT In Testing Team + Enabled the feature on /secret  | /pricing `404`  |
| NOT In Testing Team + Disabled the feature on /secret  | /pricing `404` |

## Feedback I'm looking for:
- Can you create a subscription with test data?
- If you have an active subscription, does the CTA on /pricing say "Manage Your Subscription"?

### CODE REVIEW

File: src/presenters/pages/about/pricing.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  const subscriptionStatus = useSubscriptionStatus();
+  const { createSubscriptionSession } = useAPIHandlers();
+  const stripe = useStripe();
+  const { currentUser, fetched: currentUserFetched } = useCurrentUser();
+  const userHasPufferfishEnabled = useFeatureEnabled('pufferfish');
+
+  async function subscribe() {
+    try {
+      const { data } = await createSubscriptionSession({
+        successUrl: `https://glitch.com${getUserLink(currentUser)}`,
```
Comment (sarahzinger) [inline src/presenters/pages/about/pricing.js]: might make sense to use APP_URL from our constants file here so this works nicely on staging and stuff

Comment (sarahzinger) [reply src/presenters/pages/about/pricing.js]: Also is this subscribe func the same as the last one? Might be worth thinking through if there's a place we should put this... maybe `useUserEditor` or maybe we need something like `useCustomerEditor` ? or maybe we should punt for the sake of speeding things along. Hmm I guess I've talked myself into punting lol, but something to have in the back of our minds if it's feeling like a lot of duplicated logic in the future

Comment (modernserf) [reply src/presenters/pages/about/pricing.js]: if you use `window.location.origin` it will work on remixes, too

**Pull request merged**